lib/menu.rb

```diff
@@ -0,0 +1,11 @@
+class Menu
+  attr_reader :list
+
+  def initialize
+    @list = {"Chips" => 3, "Salad" => 4, "Chicken" => 7, "Fish" => 9, "Daily special" => 15}
```

Comment:
Metrics/LineLength: Line is too long. [92/80] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)